test/modules/import-utilities.test.js

```diff
@@ -163,34 +163,37 @@ describe('Import utilities module ', () => {
     });
 
     it('Sign dataset', () => {
-        const testGraphCopy = Object.assign({}, sample_data.graph);
-        const shuffledGraphCopy = Object.assign({}, sample_data.shuffledGraph);
-        const signedOriginal = ImportUtilities.signDataset(testGraphCopy, config, web3);
+        let testGraphCopy = Object.assign({}, sample_data.graph);
+        let shuffledGraphCopy = Object.assign({}, sample_data.shuffledGraph);
+
+        testGraphCopy = Utilities.sortObjectRecursively(testGraphCopy);
+        shuffledGraphCopy = Utilities.sortObjectRecursively(shuffledGraphCopy);
 
+        const signedOriginal = ImportUtilities.signDataset(testGraphCopy, config, web3);
         const signedShuffled = ImportUtilities.signDataset(shuffledGraphCopy, config, web3);
 
         assert(signedOriginal.signature != null);
         assert(signedShuffled.signature != null);
 
         assert.equal(
-            ImportUtilities
-                .sortStringifyDataset(signedOriginal),
-            ImportUtilities
-                .sortStringifyDataset(signedShuffled),
+            JSON.stringify(signedOriginal),
+            JSON.stringify(signedShuffled),
         );
     });
 
     it('Verify dataset signature', async () => {
-        const testGraphCopy = Object.assign({}, sample_data.graph);
-        const shuffledGraphCopy = Object.assign({}, sample_data.shuffledGraph);
+        let testGraphCopy = Object.assign({}, sample_data.graph);
```

Comment:
Since the sign function now is sorting dependant I don't think we need to test if it works on a shuffled graph (we know sorting works from a previous test)